client/js/jsonedit.js

```diff
@@ -839,7 +839,6 @@ function jeAddWidgetPropertyCommands(object) {
   for(const property in object.defaults)
     if(property != 'typeClasses')
       jeAddWidgetPropertyCommand(object, property);
-  object.applyRemove();
```

Comment:
Curious what this has to do with this PR.

Reply:
:smile: 

The only change I actually wanted to make is the one in `statemanaged.js`. The problem is that that change causes the DOM element to not be attached inside the constructor anymore. And calling `applyRemove` throws an exception if the element hasn't been attached yet.

So I had to fix three problems that arose due to that change. One of them was the JSON editor creating and removing temporary widgets. I couldn't find anything in `applyRemove` that was actually necessary so I just removed it. I hope it doesn't cause any issues.

Reply:
Yeah, I see that the key changes are in `statemanaged.js` (and related change in `card.js`). I'm that changing `editmode.js` was also because the DOM element attachment got moved as well (in any case the change to `editmode.js` seems like better practice).

Reply:
Feels like all the changes are overall improvements. This one is arguably an exception. We could probably also call `object.applyInitialDelta({})` before `object.applyRemove()`. It's just unnecessary atm. :man_shrugging: